.github/workflows/code_checks.yaml

```diff
@@ -14,27 +14,27 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [8.2]
-        stability: [prefer-stable]
-        minimum-stability: [stable]
-        symfony-version: [7.0.*]
-        is-current: [false]
         include:
-          - php: '8.3'
-            symfony-version: 7.0.*
-            stability: prefer-stable
           - php: '8.4'
-            symfony-version: 7.0.*
+            symfony-version: 8.0.*
             stability: prefer-stable
+            minimum-stability: stable
+            is-current: true
           - php: '8.4'
-            symfony-version: 7.4.*
+            symfony-version: '^8.0'
             stability: prefer-stable
-            minimum-stability: dev
+            minimum-stability: RC
```

Comment:
changes in `minimum-stability` could be reverted (after the release of the stable versions)